src/tests/suite/daos_checksum.c

```diff
@@ -1540,6 +1643,7 @@ static const struct CMUnitTest csum_tests[] = {
 	CSUM_TEST("DAOS_CSUM10: Enumerate A Keys", test_enumerate_a_key),
 	CSUM_TEST("DAOS_CSUM11: Enumerate D Keys", test_enumerate_d_key),
 	CSUM_TEST("DAOS_CSUM12: Many IODs", many_iovs_with_single_values),
+	CSUM_TEST("DEDUP: ", dedup),
```

Comment:
(style)  'DEDUP' may be misspelled - perhaps 'DEDUPE'?
(style)  'dedup' may be misspelled - perhaps 'dedupe'?